sys/src/9/riscv/main.c

```diff
@@ -119,14 +120,34 @@ void bsp(void)
 
 	static Lock l; // to ensure initialization.
 	if (canlock(&l)) msg ("L can be locked\n"); else die("Can't lock L\n");
-	ilock(&l);
+	unlock(&l);
 	if (canlock(&l)) msg ("L can be NOT be locked OK\n"); else die("Can lock L after lock\n");
```

Comment:
"L can be NOT be locked OK\n" <-- I think that's got an extra 'be': how about, "L can NOT be locked OK\n"